logging/logger_manager.go

```diff
@@ -0,0 +1,53 @@
+package logging
+
+import (
+	"fmt"
+	"sync"
+)
+
+type loggerRegistry struct {
+	mu      sync.Mutex
+	loggers map[string]Logger
+	names   map[Logger]string
```

Comment:
`names` is currently unnecessary so happy to remove for now but I think we might end up in a situation where we have but not it's name (either for updating its log level or for registration, potentially in goutils) where it might be useful.

Reply:
Thanks for calling out; I think the bidirectional functionality is good to have for now.

Reply:
I'd prefer to remove this until we have a case where a reverse lookup is necessary, or if we know for sure we will need this in an upcoming change. Otherwise we just take on optional complexity by maintaining functionality we don't actually need.

Reply:
SGTM

Reply:
yeah that makes sense, it was definitely necessary for registering / manipulating zap loggers (since they were often created in contexts very far away from where their RDK logger name was defined), but I think if we are moving away from using zap loggers at all then it probably may be unnecessary